tests/UnitTests.hs

```diff
@@ -188,6 +188,8 @@ positionalTests =
         ]
     decodeWithTests =
         [ ("tab-delim", defDec { decDelimiter = 9 }, "1\t2", [["1", "2"]])
+        , ("no-escape", defDec { decEscape = NoEscape }, "\"a,b\",\"c\nd,e,f\"",
```

Comment:
If you like, one trick to make the tests a bit more readable is to write the string literals with single quotes and to have something like `replaceQuotes :: ByteString -> ByteString` to replace those with double quotes before feeding them into the parser. That eliminates the forest of backslashes.